net-vpn/eduvpn-client/eduvpn-client-4.1.3.ebuild

```
@@ -0,0 +1,38 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1
+
+if [[ ${PV} == "9999" ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/eduvpn/python-${PN}.git"
+	S="${WORKDIR}/${P}"
+else
+	SRC_URI="https://github.com/eduvpn/python-eduvpn-client/archive/refs/tags/${PV}.tar.gz -> ${P}.gh.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+	S="${WORKDIR}/python-${P}"
+fi
+
+DESCRIPTION="Linux client and Python client API for eduVPN"
+HOMEPAGE="https://www.eduvpn.org/"
+
+SLOT="0"
+LICENSE="GPL-3+"
```

Please use canonical ordering of variables.
```suggestion
LICENSE="GPL-3+"
SLOT="0"
```

---

fixed